concourse/pipelines/windows-image-build.yaml

```
@@ -161,17 +161,49 @@ jobs:
     file: publish-version/version
   - task: get-credential
     file: guest-test-infra/concourse/tasks/get-credential.yaml
+  - task: get-secret-iso
+    file: guest-test-infra/concourse/tasks/gcloud-get-secret.yaml
+    var:
+      secret_name: "winserver-20h2-64"
+  - load_var: windows-iso
+    file: gcp-secret-manager/winserver-20h2-64
+  - task: get-secret-dotnet48
+    file: guest-test-infra/concourse/tasks/gcloud-get-secret.yaml
+    var:
+      secret_name: "windows_gcs_updates_sac20h2"
+  - load_var: windows-updates
+    file: gcp-secret-manager/windows_gcs_updates_sac20h2
+  - task: get-secret-pwsh
+    file: guest-test-infra/concourse/tasks/gcloud-get-secret.yaml
+    var:
+      secret_name: "windows_gcs_pwsh"
```

syntax error i think. maybe:
```suggestion
    params:
      SECRET_NAME: "windows_gcs_pwsh"
```